index.html

```diff
@@ -436,7 +436,7 @@ <h2>Buttons and clicks: A button to add apples</h2>
         <ol>
             <li>Start with an empty unordered html list (<code>&lt;ul&gt;</code>) and a button to add apples to the list (<code>&lt;button&gt;</code>) in the static html.
                 <ol>
-                    <li>for example: <code class="start_code">&lt;ul id="list"&gt;&lt;/ul&gt;<br>&lt;button id="button"&gt;add an apple&lt;/button&gt;</code></li>
+                    <li>for example: <code class="start_code">&lt;ul id="fruits"&gt;&lt;/ul&gt;<br>&lt;button id="add-apple-button"&gt;add an apple&lt;/button&gt;</code></li>
```

Comment:
are dashed ids a good practice (I know classnames are often dashed)? I would probably underscore them, to avoid teaching identifiers with - in them

Reply:
I don't have a super strong preference - I started dashing somewhat at random, so changed a few _s to -s because I'd done most of them, but happy to change them all to _s if you'd prefer :)